dev-libs/bemenu: fix install on Prefix; fix BDEPEND

sevz17 opened the pull request.

thesamesam commented:
Could you cherry-pick https://github.com/gentoo/gentoo/pull/32520 into this PR too please and rebase on that?

sevz17 commented:
Done, I also added my sign-off and a `Closes` tag (I had to add those, right?)

Also I fixed https://bugs.gentoo.org/831031.

And finally, can you close https://bugs.gentoo.org/868750, it was fixed in this commit: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=7470ff20d0c13899856bbeedad75dcc046d93bef

sevz17 commented:
@thesamesam
